client/static/js/app.js

```diff
@@ -0,0 +1,360 @@
+window.onload = (event) => {
+d3.json('/graph').then((d)=>{
```

Comment:
Should this be:
```
d3.json(process.env.REACT_APP_GRAPH_URL).then((d)=>{
```

Using `process.env.REACT_APP_GRAPH_URL` instead of direct url `/graph`